db/pf-schema-X.Y.Z.sql

```diff
@@ -936,3 +924,15 @@ CREATE TABLE keyed (
   value LONGBLOB,
   PRIMARY KEY(id)
 ) ENGINE=InnoDB;
+
+--
+-- Table structure for table 'pf_version'
+--
+
+CREATE TABLE pf_version ( `id` INT NOT NULL PRIMARY KEY, `version` VARCHAR(11) NOT NULL UNIQUE KEY);
+
+--
+-- Updating to current version
+--
+
+INSERT INTO pf_version (id, version) VALUES (@VERSION_INT, CONCAT_WS('.', @MAJOR_VERSION, @MINOR_VERSION, @SUBMINOR_VERSION));
```

Comment:
IMO I think we should store all but use what interests us. This way it will stay consistent with pf-release, currently-at, ...